integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java

```diff
@@ -30,8 +32,20 @@ public void onTestFailure(ITestResult tr) {
         super.onTestFailure(tr);
         logUrl(tr);
         logMeasurements(tr);
-        Throwable throwable = tr.getThrowable();
-        Log.log(throwable.getMessage());
+        if (tr.getThrowable() != null) {
```

Comment:
Wouldn't it be better to put this logic whole in to our custom Logger class, and it would accept a throwable as a parameter?